digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala

```diff
@@ -226,8 +226,7 @@ object RoadAddressDAO {
           terminated = '0' and
           valid_from <= sysdate and
           (valid_to is null or valid_to > sysdate) and
-          (end_date is null or end_date > sysdate) and
-          (start_date is null or start_date <= sysdate)
+          (end_date is null or end_date > sysdate)
```

Comment:
Need to be careful, because this is not the only query that contains start_date and I think that we should test another cases where we are using start_date of road_addresses

Reply:
Checked them but didn't find anything that should compromise other functions OR needed to be removed (well except a function that was not used anywhere).

Reply:
This could return multiple rows if we have planned road_addresses in to the future. "or end_date > sysdate" must be removed. We want to see only the most recent version even though it would be in the future.